src/common/schema-validation.ts

```diff
@@ -12,22 +13,39 @@ export type JSONSchemaTypeWithId<T> = JSONSchemaType<T> & { $id: string };
  *
  * @see https://ajv.js.org/guide/managing-schemas.html#pre-adding-all-schemas-vs-adding-on-demand
  */
-export const getAjvSchema = <T>(
+const getAjvSchema = <T>(
```

Comment:
**note:** Hide Ajv integration from other modules. Currently, other modules are not aware about Ajv and rely on higher level API (e.g., `assertSchema`).